test/reporter-html/single-testid.js

```diff
@@ -0,0 +1,12 @@
+QUnit.config.testId = [ "2e48c6fa", "9ccf6855" ];
+
+QUnit.test( "Check for changed header after running filtered test", function( assert ) {
+  var html = document.getElementById( "qunit-filteredTest" ).innerHTML;
```

Comment:
Looks like our version of grunt-jscs is very much outdated. Could you look into updating that to latest? Seperate issue/PR though.